pkgs/applications/networking/nextcloud-client/default.nix

```diff
@@ -15,20 +15,22 @@
 , qtquickcontrols2
 , qtgraphicaleffects
 , sqlite
+, qtwebsockets
```

Comment:
Please move `qtwebsockets` at least one line up to keep the `qt*` dependencies together.